src/data/screenshots_de.json

```diff
@@ -55,7 +55,7 @@
                 ]
             },
             {
-                "title": "Check-In",
+                "title": "Check-in",
                 "anchor": "android_check_in",
                 "images": [
                     { "filename": "de/android/check_in_host_view_my_qr_codes", "alt": "Check In image 1 of 8" },
```

Comment:
Might consider in a future update to also update the alternative texts of the images. Same applies for iOS